### ISSUE TITLE
In-board creation of issues prompting only for title

### ISSUE THREAD

**Issue by mogul** [ZenHubIO/support#759]
In Trello, you can add a card at the bottom of a column just by clicking an "add" button at the bottom of the column, or you can create one anywhere within a column by hitting "N" while hovering the mouse over the card you want to be above the new one. You're presented with a place to just enter the title, but a flyout also allows you to add people and labels:

![image](https://cloud.githubusercontent.com/assets/47762/15624006/0bbe92f6-2432-11e6-80d4-fa456344f964.png)

Adding a description isn't available until you open the card, but when you're adding a lot of issues at once they're typically ungroomed and only have the title so far anyway. This really reduces friction!

When entering the title you can also apply labels by using hashtag completion:

![image](https://cloud.githubusercontent.com/assets/47762/15623978/aacb8620-2431-11e6-96a6-4a3c3b9c9b87.png)

**Comment by azenMatt** [comment]
Good point @mogul - we do want to make creating new issues as friction free as possible. 

a new one to try that we just released is new issue creation when you type '#' 

![image](https://cloud.githubusercontent.com/assets/7584089/16101937/9087a940-331f-11e6-8ae3-8e61bb7fa49a.png)

Hopefully this helps to quickly build sub-task lists. 

Adding directly into a pipeline is to follow :)

**Comment by mogul** [comment]
Is this intended as something you do when editing an epic, for example? It's much appreciated but I'm looking for something that works right from the board view.

**Comment by azenMatt** [comment]
> Is this intended as something you do when editing an epic, for example? It's much appreciated but I'm looking for something that works right from the board view.

Yes, more intended for while you are working with lists in an existing GH Issue 

It doesn't hit the use case of this issue, but I wanted to share incase you found it useful anyway! I'l leave this issue open so we can get something live closer to the actual use case
